CMakeLists.txt

```
@@ -2139,7 +2160,32 @@ if(QT5_TYPE STREQUAL "STATIC_LIBRARY")
 
 endif()
 
-if(UNIX AND NOT APPLE)
+
+if(APPLE)
+  if(Qt5_IS_STATIC)
+    target_link_libraries(mixxx-lib PRIVATE
+        "-weak_framework Accelerate"
+        "-weak_framework AppKit"
+        "-weak_framework AudioToolbox"
+        "-weak_framework AudioUnit"
+        "-weak_framework AVFoundation"
+        "-weak_framework CoreAudio"
+        "-weak_framework CoreFoundation"
+        "-weak_framework CoreImage"
+        "-weak_framework CoreMedia"
+        "-weak_framework CoreMidi"
+        "-weak_framework CoreServices"
+        "-weak_framework CoreVideo"
+        "-weak_framework IOSurface"
+        "-weak_framework VideoToolbox"
+    )
+  else()
+    # Used for battery measurements and controlling the screensaver on OS X and iOS.
```

```suggestion
    # Used for battery measurements and controlling the screensaver on macOS.
```
Mixxx does not support iOS.